tensorflow/lite/micro/micro_log.h

```diff
@@ -17,15 +17,21 @@ limitations under the License.
 
 #if !defined(TF_LITE_STRIP_ERROR_STRINGS)
 #include <cstdarg>
+#include <cstddef>
 // These functions can be used independent of the MicroErrorReporter to get
 // printf-like functionalitys and are common to all target platforms.
 void MicroPrintf(const char* format, ...);
 void VMicroPrintf(const char* format, va_list args);
```

Comment:
We probably should have called this MicroVprintf, MicroSnprintf and MicroVsnprintf. Lets update the latter two here, and VMicroPrintf in another PR.

Reply:
changed